src/styles/variables.css

```diff
@@ -56,6 +56,44 @@
   /* font */
   --gotham: 'Gotham', Arial, Calibri, sans-serif;
 
+  /* font size */
+
+  --font-size: 1rem;
+  --line-height: 1.5rem;
+
+  --display1: 3.75rem;
+  --display2: 3.5rem;
+  --h1: 2.25rem;
```

Comment:
Let's group the font size and line height variables together, as we generally want to know what they are relative to one another. Let's also use `font-size` and `line-height` to be more explicit.

```
--h1-font-size: 1rem;
--h1-line-height: 3.375rem;

--h2-font-size: 2rem;
--h2-line-height: 3rem;
```